src/styles/Book.css

```diff
@@ -17,7 +17,7 @@
   width: 50%;
   display: flex;
   flex-direction: column;
-  justify-content: space-evenly;
+  justify-content: space-around;
 }
 
 .percentage {
```

Comment:
- Hey, please, let's use the correct color for the image in this div to make the design close to the template as much as we can below:

*original design*
![Screenshot from 2022-11-19 05-21-07](https://user-images.githubusercontent.com/59397159/202833951-8a6ecca6-8dd9-43bf-a8d0-eb0634bcaba8.png)

*your design*
![Screenshot from 2022-11-19 05-20-24](https://user-images.githubusercontent.com/59397159/202833952-cb99cbd5-d88e-4119-bbfe-be6a9a83098d.png)


- You can use the react `<CircularProgress
          size={50}
          thickness={2}
          variant="determinate"
        />`
then install and add the `import { CircularProgress } from '@material-ui/core';`